### ISSUE TITLE
Module not found [ElementQueries, ResizeSensor, ui-tree]

### ISSUE THREAD

**Issue by carlosbridi** [angular-gantt/angular-gantt#797]
Hi! I installed the latest version of angular gantt (2.0.0-rc1) and its dependencies, I put it to run, but when I start the project (npm start), I'm having problems with the ElementQueries, ResizeSensor, and ui libs -tree.

```
ERROR in ./~/angular-gantt/dist/angular-gantt.js
Module not found: Error: Can't resolve 'ElementQueries' in 'C:\ERP\man-pcp-frontend2\node_modules\angular-gantt\dist'
 @ ./~/angular-gantt/dist/angular-gantt.js 1:2858-2883 1:3029-3159
 @ ./src/app/app.module.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./src/app/app.module.js

ERROR in ./~/angular-gantt/dist/angular-gantt.js
Module not found: Error: Can't resolve 'ResizeSensor' in 'C:\ERP\man-pcp-frontend2\node_modules\angular-gantt\dist'
 @ ./~/angular-gantt/dist/angular-gantt.js 1:2884-2907 1:3029-3159
 @ ./src/app/app.module.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./src/app/app.module.js

ERROR in ./~/angular-gantt/dist/angular-gantt.js
Module not found: Error: Can't resolve 'ui.tree' in 'C:\ERP\man-pcp-frontend2\node_modules\angular-gantt\dist'
 @ ./~/angular-gantt/dist/angular-gantt.js 1:2962-2980 1:3029-3159
 @ ./src/app/app.module.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./src/app/app.module.js
```

I am using the webpack (I do not know if there is any problem with this).

If I change the angular-gantt.js file, and put the relative path to the resizesensor, elementqueries, and ui-tree ... it works ..
Like that:
`module.exports = factory (require ("angular"), require ("moment"), require ("../../ node_modules / css-element-queries / ElementQueries"), require ("../../ node_modules / css-element-queries / ResizeSensor "), require (" angular-native-dragdrop "), require (" jsPlumb "), require (" ../../ node_modules / angular-ui-tree / ui.tree "));`

But the problem of doing this is that I have to upload the whole folder, and I did not want to, since I'm with my project in git.

i'm using angular 1.5.8

Anothers libs works fine, like jsPlumb, moment, etc..

Ps.: i'm using webpack to startup project

**Comment by hevbevt** [comment]
Try to use 'script-loader' in your webpack config file to import ResizeSensor, ElementQueries, moment and angular-gantt.  I fixed that issue in this way.
``` 
            {
                test: require.resolve('css-element-queries/src/ResizeSensor'),
                loader: 'script-loader'
            },
            {
                test: require.resolve('css-element-queries/src/ElementQueries'),
                loader: 'script-loader'
            },
            {
                test: require.resolve('moment'),
                loader: 'exports-loader?window.moment!script-loader'
            },
            {
                test: require.resolve('angular-gantt'),
                loader: 'script-loader'
            },
```

**Comment by carlosbridi** [comment]
Thx for the answer, but now, i've another problem...

Uncaught Error: [$injector:modulerr] Failed to instantiate module app due to:
Error: [$injector:modulerr] Failed to instantiate module gantt.table due to:
Error: [$injector:nomod] Module 'gantt.table' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.
http://errors.angularjs.org/1.5.8/$injector/nomod?p0=gantt.table

On load app, i declare ['gantt', 
    'gantt.table',
    'gantt.movable', 
    'gantt.progress']

Like doc, and...i no idea to fix it

**Comment by hevbevt** [comment]
I updated my angular-gantt project with webpack3 [angular-gantt-webpack](https://github.com/hevbevt/angular-gantt-webpack) hope it can help u.

**Comment by colinwurtz** [comment]
@carlosbridi did you ever get this figured out? Getting the same error myself

**Comment by carlosbridi** [comment]
@colinwurtz No, I gave up using version 2.0.0rc1 and I'm using 1.3. My project is in version 1 of the Angular.

**Comment by Toilal** [comment]
I'm sorry about that. I don't have time to maintain the project, sadly :(

**Comment by colinwurtz** [comment]
@carlosbridi - I'm also working on an existing project that uses angular 1.x . I was able to pull in version 1.3 and it's playing nice with webpack now. Thanks for your help!

@Toilal No worries - I know how it goes. Thanks for a great library!
